sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinitionTest.java

```diff
@@ -51,17 +56,27 @@ public void test() {
 
   @Test
   public void test_before_9_3() {
-    SonarRuntime sonarRuntime = SonarRuntimeImpl.forSonarQube(Version.create(9, 2), SonarQubeSide.SCANNER, SonarEdition.COMMUNITY);
-    AbstractRulesDefinition sut = new TestRulesDefinition(sonarRuntime);
-    RulesDefinition.Context context = new RulesDefinition.Context();
-    sut.define(context);
+    assertThat(getSecurityStandards(Version.create(9, 2), "S1111")).containsExactlyInAnyOrder("cwe:117", "cwe:532", "owaspTop10:a10", "owaspTop10:a3");
+  }
 
-    RulesDefinition.Repository repository = context.repository("test");
-    assertThat(repository).isNotNull();
+  @Test
+  public void test_security_standards_9_4_PCI_DSS_is_not_available() {
+    assertThat(getSecurityStandards(Version.create(9, 4), PCI_DSS_RULE_KEY)).containsExactlyInAnyOrder();
+  }
 
-    RulesDefinition.Rule rule = repository.rule("S1111");
-    assertThat(rule).isNotNull();
-    assertThat(rule.securityStandards()).containsExactlyInAnyOrder("cwe:117", "cwe:532", "owaspTop10:a10", "owaspTop10:a3");
+  @Test
+  public void test_security_standards_9_5_PCI_DSS_is_available() {
+    assertThat(getSecurityStandards(Version.create(9, 5), PCI_DSS_RULE_KEY)).containsExactlyInAnyOrder("pciDss-3.2:6.5.10", "pciDss-4.0:6.2.4");
+  }
+
+  @Test
+  public void test_security_standards_9_8_ASVS_is_not_available() {
+    assertThat(getSecurityStandards(Version.create(9, 8), OWASP_ASVS_RULE_KEY)).containsExactlyInAnyOrder();
```

Comment:
This assertion is not clear. It should be something like `isEmpty`, shouldn't it?

Reply:
Same here